packages/now-cli/src/util/dev/types.ts

```diff
@@ -27,11 +27,12 @@ export interface DevServerOptions {
   devCommand?: string;
   frameworkSlug?: string;
   projectSettings?: ProjectSettings;
+  environmentVars?: Env;
 }
 
 export interface EnvConfigs {
   /**
-   * environment variables from `.env.build` file (deprecated)
```

Comment:
I'm curious why deprecated was removed?

Reply:
It's no longer deprecated. @TooTallNate and I discussed on Slack a while back